sdk/nodejs/tests/automation/localWorkspace.spec.ts

```diff
@@ -21,10 +21,8 @@ import {
     EngineEvent,
     fullyQualifiedStackName,
     LocalWorkspace,
-    OutputMap,
-    ProjectSettings,
+    OutputMap, parseAndValidatePulumiVersion, ProjectSettings,
     Stack,
-    parseAndValidatePulumiVersion,
```

Comment:
Nit: Why did these lines need to change?

Reply:
just vscode reording imports, I can keep it as is as lone as I don't save in vsc.